gradle/libs.versions.toml

```diff
@@ -15,10 +16,10 @@ androidXNavigation = "2.3.0"
 androidXRecyclerView = "1.1.0"
 androidXWorkManager = "2.7.0"
 androidXConstraintLayout = "2.0.4"
-androidXComposeRuntime = "1.0.5"
+androidXComposeRuntime = "1.1.0-rc01"
```

Comment:
is this required for this PR ?

Reply:
Yes, Compose kotlin dependency needs to be exactly the same as the kotlin dependency we use